src/applications/claims-status/components/ClaimLetterListItem.jsx

```diff
@@ -36,7 +36,8 @@ const filename = 'ClaimLetter.pdf';
 const downloadHandler = () => {
   recordEvent({
     event: 'claim-letters-download',
-    'click-text': 'Download letter (PDF)',
+    eventLabel: `${environment.API_URL}/v0/claim_letters/:id`,
```

Comment:
Should this use `${id}` here?

Reply:
I'm hoping to obfuscate the id because it is coming through in GA right now
<img width="914" alt="Screen Shot 2023-01-12 at 3 28 15 PM" src="https://user-images.githubusercontent.com/13838621/212185307-eb3d6d25-e77e-42ef-bae1-ca0f4934fa1a.png">